specifyweb/specify/models.py

```diff
@@ -7617,10 +7617,13 @@ class Meta:
         ordering = ()
         # unique_together = (('childcog'), ('childco'))
         constraints = [
+            # Constraint so that childcog and childco are not both null in the same record
             CheckConstraint(
-                check=~Q(childcog__isnull=True, childco__isnull=True),
+                check=models.Q(
+                    ("childco__isnull", True), ("childcog__isnull", True), _negated=True
```

Comment:
Does that make an eclusive or?

Something like `~(Q(childco__isnull=True) ^ Q(childcog__isnull=True))` should be [enough](https://github.com/django/django/blob/6b3f55446fdc62bd277903fd188a1781e4d92d29/django/db/models/query_utils.py#L77).

But, you might be better off adding a business rule for it (so it can be displayed nicely)